man/systemd-sbsign.xml

```diff
@@ -60,8 +60,8 @@
       <varlistentry>
         <term><option>--output=<replaceable>PATH</replaceable></option></term>
 
-        <listitem><para>Specifies the path where to write the signed PE binary or the data to be signed
-        offline when using the <option>--prepare-offline-signing</option> option.</para>
+        <listitem><para>Specifies the path where to write the signed output or the data to be signed offline
```

Comment:
hmm, i think man page docs should be precise on these things. Please enumerate possible outputs here, i.e. keep PE binary in place, but also list possible other outputs.